projects/kubernetes/kubernetes/build/lib/images.sh

```diff
@@ -133,3 +133,48 @@ function build::images::docker::push(){
         docker push ${repository}/${repo_prefix}/${binary}:${image_tag}
     done
 }
+
+# Create platform-specific OCI image tars
+function build::images::pause_tar(){
+    local -r go_runner_image="$1"
+    local -r version="$2"
+    local -r image_tag="$3"
+    local -r context_dir="$4"
+    local -r bin_dir="$5"
+    local -r skip_arm="$6"
+
+    for platform in "${KUBE_LINUX_IMAGE_PLATFORMS[@]}"; do
+        if [ "$platform" == "arm64" ] && [ $skip_arm == true ]; then
```

Comment:
arm builds for pause fail on the presubmit cluster, i believe because we dont have qemu setup fully on the fargate nodes.  Skipping arm in this case is what we did before, but now we build both amd64 and arm64 in post submit.